openedx/core/djangoapps/xmodule_django/models.py

```diff
@@ -158,6 +149,17 @@ def run_validators(self, value):
         return super(OpaqueKeyField, self).run_validators(value)
 
 
+# This overrides the default __isnull model filter to help enforce the special way
+# we handle null / empty values in OpaqueKeyFields.
+class OpakeyKeyFieldEmptyLookupIsNull(IsNull):
+    def get_prep_lookup(self):
+        raise TypeError('Use this models .Empty member rather than None or __isnull '
```

Comment:
"models" -> "model's"

Reply:
shouldn't this say "Use this **field's** .Empty member..."?